### ISSUE TITLE
`real()` and `int64()` producing incorrect values

### ISSUE THREAD

**Issue by tinkerer-red** [YoYoGames/GameMaker-Bugs#8013]
### Description

Now to be fair this might be intentional, in monthly as changes have been made recently with #7685 
But as of monthly:

inputting a string into `real` or `int64` which contains underscores (something supported in the ide for easier to read numbers) will produce difference results then it would with out those underscores

### Expected Change

It could go either way honestly, but it could either throw an error, or strip the underscores to fit the standards from the language

### Steps To Reproduce

```gml
show_debug_message(0xFFFF_FFFF_1234_5678) // -3989547400
show_debug_message(0xFFFFFFFF12345678) // -3989547400


show_debug_message("\n")
var _val = "0xFFFF_FFFF_1234_5678"
show_debug_message(real(_val)) // 65535
show_debug_message(int64(_val)) // -9223372036854753672

show_debug_message("\n")
var _val = "0xFFFFFFFF12345678"
show_debug_message(real(_val)) // 18446744069720004608.00
show_debug_message(int64(_val)) // 9223372036854775807

show_debug_message("\n")
//show_debug_message(real("0xFFFF_FFFF_1234_5678")) // Cant compile
show_debug_message(int64("0xFFFF_FFFF_1234_5678")) // -9223372036854753672

show_debug_message("\n")
show_debug_message(real("0xFFFFFFFF12345678")) // -3989547400
show_debug_message(int64("0xFFFFFFFF12345678")) // 9223372036854775807
```

### How reliably can you recreate this issue using your steps above?

Always

### Which version of GameMaker are you reporting this issue for?

2024.8.1 (Monthly)

### Which operating system(s) are you seeing the problem on?

Windows 10

### Are you running GameMaker from inside your Steam library?

No

### Sample Package Attached?

- [X] I have attached my bug package

### Sample Project Added?

- [X] I have included a small sample project

**Comment by tinkerer-red** [comment]
This is also a cross post with #8011 which goes into a more specific issue.

**Comment by rwkay** [comment]
Fixed in 2024.11

**Comment by sihammill** [comment]
Fix verified in IDE v2024.1100.0.653  Runtime v2024.1100.0.679

**Comment by jackerley** [comment]
GMRT fails to convert string 
```
Unhandled exception: Error: unable to convert string "0xFFFF_FFFF_1234_5678" to number
```
